### PR TITLE
fix: update AuthorizableUser interface

### PR DESCRIPTION
**Pull request by mashirozx**
fix: #221

### CODE REVIEW

**Review by liquidautumn — APPROVED** [review]

**Comment by mashirozx** [comment]
Oh no,  I have no idea about this type chain 😢

File: src/interfaces/authorizable-user.interface.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export interface AuthorizableUser<Roles = string, Id = string> {
+export interface AuthorizableUser<Roles = string, Id = string | number> {
```
Comment (nguyenthenguyen) [inline src/interfaces/authorizable-user.interface.ts:1]: @mashirozx @liquidautumn Can we change `Id` to `any` type?

```
export interface AuthorizableUser<Roles = string, Id = any> {
  id: Id;
  roles: Array<Roles>;
}

```

**Review by liquidautumn — CHANGES_REQUESTED** [review]
Research required

**Comment by ppodds** [comment]
The problem is still there. Has any turnaround?

**Comment by rubiin** [comment]
@liquidautumn any updates on this. Seems to be a blocking issue for nosql dbs where the id is mostly string

**Comment by liquidautumn** [comment]
Bug fixed in #534

**Pull request closed without merging**